graphql-dgs-codegen-core/src/test/kotlin/com/netflix/graphql/dgs/codegen/CodeGenTest.kt

```diff
@@ -462,31 +454,33 @@ class CodeGenTest {
         // schema contains nullable, non-nullable and list types as interface fields  and fields that are
         // not interfaces
         val schema = """
-            interface Pet {
-                id: ID!
-	            name: String
-                address: [String!]!
-                mother: Pet!
-                father: Pet
-            	parents: [Pet]
-             }
-            type Dog implements Pet {
-                id: ID!
-	            name: String
-                address: [String!]!
-                mother: Dog!
-                father: Dog
-            	parents: [Dog]
-            }
-            type Bird implements Pet {
-                id: ID!
-	            name: String
-                address: [String!]!
-                mother: Bird!
-                father: Bird
-            	parents: [Bird]
-            }
-        """.trimIndent()
+            |interface Pet {
```

Comment:
There were plenty of cases where we were mixing tabs and spaces and the Kotlinter plugin was now failing the build.